plugins/ScheduledReports/API.php

```diff
@@ -86,18 +87,22 @@ public function __construct(LoggerInterface $logger)
      * @param array $reports array of reports
      * @param array $parameters array of parameters
      * @param bool|int $idSegment Segment Identifier
+     * @param bool $evolutionPeriodFor If true, evolution graphs cover each day within the period. If false, evolution graphs
+     *                                 cover the previous N periods.
+     * @param int|null $evolutionPeriodN The previous N periods to query in evolution graphs if $isEvolutionGraphEachDay is true.
      *
      * @return int idReport generated
      */
-    public function addReport($idSite, $description, $period, $hour, $reportType, $reportFormat, $reports, $parameters, $idSegment = false)
+    public function addReport($idSite, $description, $period, $hour, $reportType, $reportFormat, $reports, $parameters, $idSegment = false, $evolutionPeriodFor = 'prev',
```

Comment:
`$evolutionPeriodFor = 'prev'` doesn't match the php docs where it says a boolean?